rpm/src/main/java/org/eclipse/packager/rpm/VerifyFlags.java

```diff
@@ -0,0 +1,54 @@
+package org.eclipse.packager.rpm;
+
+/**
+ * Constants to identify the RPM verification flags.
+ * See https://github.com/ctron/rpm-builder/issues/41
+ * The name of this enum is questionable.
+ * It should rather be "VerifyFlag" (singular), but I leave it this way, cf. {@link FileFlags}.
+ * The constants and their value are from http://ftp.rpm.org/api/4.14.0/group__rpmvf.html
+ * @since 0.15.2
+ * @author Oliver Matz
```

Comment:
The project doesn't use `@author` tags. If you want your name added somehow, I can understand and we could achieve this using the NOTICE file instead.

Reply:
fixed + pushed.